src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("2.2.0.4207")]
+[assembly: AssemblyVersion("2.3.0.5157")]
```

Comment:
this file should not be committed.

Reply:
Yes, I already added it into git ignore.